frameit/lib/frameit/screenshot.rb

```diff
@@ -9,7 +9,7 @@ class Screenshot
     # path: Path to screenshot
     # color: Color to use for the frame
     def initialize(path, color)
-      UI.user_error "Couldn't find file at path '#{path}'" unless File.exist? path
+      UI.user_error! "Couldn't find file at path '#{path}'" unless File.exist? path
```

Comment:
While you're at it, can you surround the actual error message with `("...")`? There is no rubocop rule for it unfortunately.